### DISCUSSION TITLE
Localization in french

### DISCUSSION THREAD

**Discussion by apohutenumera** [fabianwennink/IconCaptcha-Plugin-jQuery-PHP#50] [Q&A]
Hi, is the package translated in french ? or only in english for frontend ?
Regards

**Comment by fabianwennink** [comment]
Hi, sorry for the late response. The only language supported by default is English, however you are free to translate the messages in your own implementation. More details about how to do that can be found here: https://github.com/fabianwennink/IconCaptcha-PHP/wiki/Localization
